### ISSUE TITLE
code and dataset release

### ISSUE THREAD

**Issue by hmfann** [balintio/nppd#2]
Hello, thanks for your impressive and inspiring work of Monte Carlo denoising! Do you have any plan to release your code and dataset to the community?

**Comment by balintio** [comment]
Hi, thanks for the kind words! Due to popular request (and for our following projects), we started porting our code to PyTorch. Unfortunately, I got busy with another submission, but now I have some time to finish this. Everything should be up next week. Apologies!

**Comment by hmfann** [comment]
That would be very nice! Thanks for your work!

**Comment by balintio** [comment]
#3 

I'm very sorry. Unfortunately, I've been on sick leave for six weeks. I had intended to finish the code release before then, but I got delayed by other matters. I'm getting around to it now.

**Comment by hmfann** [comment]
It's not too late, and remember to take good care of yourself buddy. Looking forward to the dataset and code :)

**Comment by mchenwang** [comment]
Wish you a speedy recovery.

**Comment by CzzzzH** [comment]
Would it be possible for the code before SIGGRAPH submission due?

**Comment by xinrui-zhuang** [comment]
watting for code

**Comment by yy6768** [comment]
Hello, I‘m sorry to bother you. I was just wondering that if it could be possible for the code before 2024 April.

**Comment by balintio** [comment]
Thank you for the good wishes! I had to have another surgery in February, but thankfully, my recovery was smoother this time. I should be all right health-wise for the foreseeable future.

I just verified that our reimplementation works and reproduced some previous results practically within run-to-run variation. The old code was a mess, so this should be much more helpful. I'm running the final validations on our cluster as I write this, and the rest of the team will review the documentation later in the week. I can finally confirm the release later this week.

We've started a collaboration titled ["Perception-Guided Neural Monte Carlo Sampling and Reconstruction"](https://saarland-informatics-campus.de/en/piece-of-news/chip-manufacturer-intel-and-the-state-government-of-saarland-support-computer-science-research-projects/). This collaboration will include projects similar to NPPD, with a strong emphasis on open science. The new implementation of NPPD will serve as a starting point, not just for us but anyone who would like to contribute. 

We're sharing a framework for using our (and any contributors') datasets, running evaluations, and maintaining benchmarks for easy comparisons. The new NPPD implementation is based on this framework, plus we have some documentation. I'll be more active after the public release this week should any questions or initiatives for collaboration/contribution arise.
